Bug - core IR error message (#35560)
demisto/content: Packs/Core/ReleaseNotes/3_0_51.md

#### Integrations

##### Investigation & Response

> - Fixed an issue where an error returned instead of an entry when using the **blocklist_files_command** and **allowlist_files_command** commands when all of the hashes had already been added to the block or allow list.

Reviewer: ```suggestion
- Fixed an issue in the ***core-blocklist-files*** and ***core-allowlist-files*** commands where an error was returned when all of the hashes had already been added to the blocklist or the allowlist.
```
